Vagrantfile

```diff
@@ -9,7 +9,7 @@ Vagrant.configure("2") do |config|
 
 	# The following will not work on case-insensitive filesystems like those provided by
 	# Windows and macOS, as the Linux kernel source tree has files that differ only by case.
-        unless Vagrant::Util::Platform.fs_case_sensitive?
+        if Vagrant::Util::Platform.fs_case_sensitive?
```

Comment:
Now it's backwards  :confused: 
`unless` == `if not`
Right now, your logic is "if platform is case sensitive, enable the synced folder".
You might want to change it back to `unless`, or to `if not` if it makes your more comfortable

Reply:
It's ok, it requires the fs to be case-sensitive to work